/**
 * Error thrown when an assertion fails.
 *
 * @example
 * ```ts
 * import { AssertionError } from "@std/assert/assertion-error";
 *
 * throw new AssertionError("Assertion failed");
 * ```
 */ export declare class AssertionError extends Error {
  /** Constructs a new instance. */ constructor(message: string);
}
//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImFzc2VydGlvbl9lcnJvci50cyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFHQTs7Ozs7Ozs7O0NBU0MsR0FDRCxPQUFPLGNBQU0sdUJBQXVCO0VBQ2xDLCtCQUErQixHQUMvQixZQUFZLFNBQVMsTUFBTTtBQUk3QiJ9